include/cellular_config_defaults.h

```diff
@@ -146,4 +146,82 @@
+ * messages.
+ *
+ * To enable error level logging in the cellular library, this macro should be mapped to the
+ * application-specific logging implementation that supports error logging.
+ *
+ * @note This logging macro is called in the cellular library with parameters wrapped in
+ * double parentheses to be ISO C89/C90 standard compliant. For a reference
+ * POSIX implementation of the logging macros, refer to cellular_config.h files.
+ *
+ * <b>Default value</b>: Error logging is turned off, and no code is generated for calls
+ * to the macro in the cellular library on compilation.
+ */
+#ifndef LogError
+    #define LogError( message )
+#endif
+
+/**
+ * @brief Macro that is called in the cellular library for logging "Warning" level
+ * messages.
+ *
+ * To enable warning level logging in the cellular library, this macro should be mapped to the
+ * application-specific logging implementation that supports warning logging.
+ *
+ * @note This logging macro is called in the cellular library with parameters wrapped in
+ * double parentheses to be ISO C89/C90 standard compliant. For a reference
+ * POSIX implementation of the logging macros, refer to cellular_config.h files.
+ *
+ * <b>Default value</b>: Warning logs are turned off, and no code is generated for calls
+ * to the macro in the cellular library on compilation.
+ */
+#ifndef LogWarn
+    #define LogWarn( message )
+#endif
+
+/**
+ * @brief Macro that is called in the cellular library for logging "Info" level
+ * messages.
+ *
+ * To enable info level logging in the cellular library, this macro should be mapped to the
+ * application-specific logging implementation that supports info logging.
+ *
+ * @note This logging macro is called in the cellular library with parameters wrapped in
+ * double parentheses to be ISO C89/C90 standard compliant. For a reference
+ * POSIX implementation of the logging macros, refer to cellular_config.h files.
+ *
+ * <b>Default value</b>: Info logging is turned off, and no code is generated for calls
+ * to the macro in the cellular library on compilation.
+ */
+#ifndef LogInfo
+    #define LogInfo( message )
+#endif
+
+/**
+ * @brief Macro that is called in the cellular library for logging "Debug" level
+ * messages.
+ *
+ * To enable debug level logging from cellular library, this macro should be mapped to the
+ * application-specific logging implementation that supports debug logging.
+ *
+ * @note This logging macro is called in the cellular library with parameters wrapped in
+ * double parentheses to be ISO C89/C90 standard compliant. For a reference
+ * POSIX implementation of the logging macros, refer to cellular_config.h files.
+ *
+ * <b>Default value</b>: Debug logging is turned off, and no code is generated for calls
+ * to the macro in the cellular library on compilation.
+ */
+#ifndef LogDebug
+    #define LogDebug( message )
+#endif
+
+
+#define CellularLogError    LogError
```

Comment:
Suggest we use LogError in the code and remove CellularLogError.
The same suggestion for CellularLogWarn, CellularLogInfo, CellularDebug.

Reply:
That sounds good.

Reply:
please refer to this commit [[Cellular Library] Remove all prefix Cellular in Log functions.](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/34/commits/5d67a099521cb3b75736c56431f04db34a2ce86d)